workspace/utils/blocks.py

```diff
@@ -6,3 +6,27 @@ def get_text_block(text, block_type="section", text_type="mrkdwn"):
             "text": text,
         },
     }
+
+
+def get_header_block(header_text, text_type="plain_text"):
+    return get_text_block(header_text, block_type="header", text_type=text_type)
+
+
+def get_basic_header_and_text_blocks(
+    header_text: str, texts: list | str, text_type="mrkdwn"
+):
+    """
+    Returns a list of blocks: a header block and text blocks with little room for customization.
+    "texts" can be a list of strings or a single string; In the former case, each string will be a separate text block.
+    The header block is a plain text header block.
+    The text blocks have block type "section" and the text type is "mrkdwn" by default.
+    """
+    header_block = get_header_block(header_text)
+    if isinstance(texts, str):
+        texts = [texts]
+    if len(texts) > 49:
```

Comment:
I think this also isn't covered by tests; given that the only place I've seen a message exceed 50 blocks is the funding report, and that handles it already, I think you'd be fine to `pragma: no cover` this one

Reply:
Oops - completely forgot about test coverage here, have pragma: no cover-ed